lib/backend/k8s/k8s_fv_test.go

```diff
@@ -747,7 +747,7 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 	}()
 
 	It("should not error on unsupported List() calls", func() {
-		objs, err := c.List(model.BGPPeerListOptions{})
+		objs, err := c.List(model.NodeBGPPeerListOptions{})
```

Comment:
Ah I see what you mean.  I'll do this first thing as part of the PR.  On my today list.